Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+    agent any
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '1'))
+        timestamps()
+    }
+    parameters {
+        string(name: 'CONFIG_REPOSITORY', defaultValue: 'https://gitlab.freifunk-kitzingen.de/derhoffmann/site-ffvec.git', description: 'The repository with the configuration')
+        string(name: 'CONFIG_BRANCH', defaultValue: 'master', description: 'The branch with the configuration')
+        string(name: 'GLUON_VERSION', defaultValue: 'master', description: 'Gluon version')
+        choice(name: 'GLUON_DEBUG', choices: ['1', '2'], description: 'Build with -j1 and V=s for debugging build errors')
```

Comment:
It must be zero or one.

See https://gluon.readthedocs.io/en/latest/user/getting_started.html#special-variables